### PR TITLE
Select all text when overriding score / total score

### PR DESCRIPTION
**Pull request by alogan519**

### CODE REVIEW

File: d2l-rubric-editable-score.html [hunk 1]
```diff
@@ -126,6 +126,10 @@
 			focus: function() {
 				var elem = Polymer.dom(this.root).querySelector('d2l-input-text');
```
Comment (mpharoah-d2l) [inline d2l-rubric-editable-score.html]: I think you can just use `var elem = this.$['text-area']` to get this

Comment (alogan519) [reply d2l-rubric-editable-score.html]: so I think here I am looking for the d2l-input-text as there are some custom focus handling that happens (although probably would work your way as well).

Comment (mpharoah-d2l) [reply d2l-rubric-editable-score.html]: Yeah, and it has id 'text-area' right? using `this.$.<id>` should give you the element with local ID <id>, so `this.$['text-area']` is equivalent to `Polymer.dom(this.root).querySelector('#text-area')`, but more efficient

Comment (mpharoah-d2l) [reply d2l-rubric-editable-score.html]: There's only one `d2l-input-text` and it has id `text-area`, so they should be exactly equivalent

Comment (alogan519) [reply d2l-rubric-editable-score.html]: oh ok I see what you are saying now.

**Pull request merged**